samples/spring-security-xsuaa-usage/src/main/java/sample/spring/xsuaa/SecurityConfiguration.java

```diff
@@ -60,8 +66,8 @@ protected void configure(HttpSecurity http) throws Exception {
 	 * Customizes how GrantedAuthority are derived from a Jwt
 	 */
 	Converter<Jwt, AbstractAuthenticationToken> getJwtAuthenticationConverter() {
-		TokenAuthenticationConverter converter = new TokenAuthenticationConverter(xsuaaServiceConfiguration);
-//		converter.setLocalScopeAsAuthorities(true);
+		TokenAuthenticationConverter converter = new TokenAuthenticationConverter(xsuaaServiceConfiguration.getAppId());
```

Comment:
why not using TokenAuthenticationConverter converter = new TokenAuthenticationConverter(xsuaaServiceConfiguration);

Reply:
Honestly, I would like to get rid of  `TokenAuthenticationConverter(XsuaaServiceConfiguration)` , as only the appId is required here....